README.md

```diff
@@ -6,5 +6,12 @@ A set of Netuitive analytics configurations, polices, dashboards, and reports th
 
 ### Version next
 
+### Version 1.0.1
```

Comment:
```suggestion
```
Versions shouldn't be set in PRs, so this should still be Version next.

Reply:
@Grumblesmith Done - I moved the description back to the Version next. Thanks.